lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -60,11 +60,22 @@ def supports_scrubbing?
       end
 
       def scrub(transcript)
-        transcript
+        clean_transcript = remove_invalid_utf_8_byte_sequences(transcript)
+
+        clean_transcript.
+          gsub(%r((Authorization: Gateway )(.*)(:)), '\1[FILTERED]\3').
+          gsub(%r((<username>)(.*)(</username>)), '\1[FILTERED]\3').
```

Comment:
Is it necessary to scrub out the username? Looking at another gateway, the username was left in.

Reply:
I don't think there is any particular reason it needs to be scrubbed. Unscrubbed it.